velox/connectors/hive/storage_adapters/abfs/AbfsReadFile.h

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <azure/storage/blobs.hpp>
```

Comment:
Consider removing this header and forward declaring any classes.

Reply:
hey @majetideepak  you mean replace this general hpp to any specific hpp which defines the class used ?
for example ```#include <azure/storage/blobs/blob_client.hpp>``` ?